testing/it/e2e-selenium-utils/src/main/java/com/adobe/cq/wcm/core/components/it/seljup/util/components/text/BaseText.java

```diff
@@ -14,7 +14,7 @@
  ~ limitations under the License.
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
-package com.adobe.cq.wcm.core.components.it.seljup.components.text;
+package com.adobe.cq.wcm.core.components.it.seljup.util.components.text;
 
```

Comment:
@dhavkuma 
can you update the following methods?
`
    public boolean isTextRendered(String textValue) {
        if($(rendered).isDisplayed()) {
            return $(rendered).innerHtml().trim().equals(textValue);
        }
        return false;
    }
`
and 

`   public boolean isTextRenderedWithXSSProtection(String textValue) {
        if($(editorConf).isDisplayed()) {
            return $(editorConf).innerHtml().trim().equals(textValue);
        }
        return false;
    }`